airbyte-integrations/connectors/source-twilio-taskrouter/source_twilio_taskrouter/spec.yaml

```diff
@@ -0,0 +1,25 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/twilio-taskrouter
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Twilio Taskrouter Spec
+  type: object
+  required:
+    - account_sid
+    - auth_token
+    - sid
+  additionalProperties: true
+  properties:
+    # 'TODO: This schema defines the configuration required for the source. This usually involves metadata such as database and/or authentication information.':
+    account_sid:
+      type: string
+      description: Twilio Account ID
+      airbyte_secret: true
+    auth_token:
+      type: string
+      description: Twilio Auth Token
+      airbyte_secret: true
+    sid:
+      type: string
+      description: Workspace ID
+      airbyte_secret: true
```

Comment:
can you explain why the workspace id is a secret?

Reply:
should it not be if it is not sensitive info?

Reply:
> sid -- The unique string that we created to identify the Workspace resource.

It's not a PII data.

Reply:
sure, i'll change it